Automattic/jetpack
# Photon: do not use Photon on private simple sites

Follow-up to #19201

## Proposed changes:

We should not use Photon when creating Tiled Galleries on private WordPress.com Simple sites (such sites can now use a mapped domain, and not just the ignored `*.files.w0rdpress.com` subdomain.

### Other information:


## Jetpack product discussion

* p58i-gPS-p2#comment-62620
* p1712343794015979-slack-C02FMH4G8

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

This can be tested on a WordPress.com Simple site with this patch applied (see the comment below to learn how to apply the patch to your WordPress.com sandbox).

Your simple site must be private, and you should have added a mapped domain starting with a letter between a and f, and that domain must be the primary domain on the site.

* Go to Posts > Add New
* Add a Tiled Gallery block
* Upload images to the block
* Publish your post
* Access the post on the frontend
    * The images in the gallery should be properly displayed and should not use a `i*.wp.com` domain.
* You can try the same thing with other image types: a single image block ; an image inserted inside a classic block ; a tiled gallery created inside a classic block.
